package.json

```diff
@@ -53,7 +53,7 @@
   },
   "dependencies": {
     "@patternfly/patternfly": "2.71.6",
-    "@patternfly/react-charts": "5.3.22",
+    "@patternfly/react-charts": "6.1.0",
```

Comment:
Bump a major version of react-charts may have more implications.
Have you tested in deep to see if there is no regression in the components that use this library ?

Reply:
About this...I prefer to go a patternfly general update...sometimes react-charts use another lib of patternfly like dependency, we should upgrade them to the "next core release" instead only one I think

Reply:
Yes.
A major upgrade on patternfly can't be changed in a quick PR.
It requires a major test by all sides Dev + QE to be sure that nothing (features and style) are broken.

So, I don't want to open that task yet.